docs/boards/fysetc/spider-11.md

```diff
@@ -0,0 +1,214 @@
+release as it's a latching style SD Card socket).
+
+For more information; this is documented on the Fysetc pages:
+https://github.com/FYSETC/FYSETC-SPIDER#42-Klipper
+
+!> If you have an ADXL345 connected to your Spider as shown below, this
+must be disconnected from your Spider to avoid conflicts with the SD
+Card while loading the firmware.
+
+### via SSH(PuTTY)/USB
+
+While the SD Card method described above is the easiest, users that are
+comfortable with SSH/PuTTY may prefer this more advanced method.
+
+Fysetc provide instructions on installing Klipper here:
+https://github.com/FYSETC/FYSETC-SPIDER#42-Klipper but some parts of
+that are less clear than one might wish so here is the sequence that
+worked for the author.
+
+Make sure your board is connected to the Pi (USB-C on the Spider, USB-A
+on the Pi).  Connect with SSH (PuTTy for Windows users) to the Pi (login
+pi, password raspberry if you did not change the defaults).
+
+![Fysetc Spider V1.1 BT0 Jumper](_media/BT0-jumper.png)
+
+Connect a jumper between 3.3V and BT0 on the Spider as shown above.
+
+Press the reset button on the Spider.
+
+On the Pi, run the following command:
+
+	dfu-util --list
+
+You should see a list of DFU devices.  This is your Spider ready to have
+the firmware uploaded.
+
+Build the firmware which is covered here: https://rat-rig.github.io/V-CoreOS/#/manual-firmware-compilation
+
+Once the firmware is built on the Pi run:
+
+	sudo service klipper stop
+	dfu-util -a 0 -s 0x08000000:leave -D ~/klipper/out/klipper.bin
+
+You should see the firmware being written to your Spider.
+
+Now remove the jumper between 3.3V and BT0 on the Spider.  Press the reset button on the Spider.
+
+Run "lsusb" on the Pi and you should see a device by the name "OpenMoko, Inc.". This is your Spider running Klipper.
+
+run the command "sudo service klipper start". V-CoreOS should now be able to communicate with your Spider.
+
+### Klipper Updates
+
+Sometimes klipper makes changes to the microcontroller code and thus
+your Spider need to be reflashed with new firmware.  You can do that in 2
+ways.
+
+The updates can be installed with an SD Card (same method as
+the first install described above) or over a Pi/Spider USB connection from V-CoreOS
+using SSH(PuTTY) and the provided
+~/klipper_config/config/boards/fysetc-spider/make-and-flash-mcu.sh
+command so you don't need to fiddle around with SD Cards.
+
+## Setup
+
+If you're going through initial setup please continue in the [installation guide](installation.md#setup)
+
+## ADXL345 Connection
+
+In your printer.cfg add the following:
+
+	#   For ADXL345
+	[resonance_tester]
+	accel_chip: adxl345
+	# Change the following to the centre of your bed.
+	probe_points: 200,200,20
+
+Connect the ADXL345 to the Spider like so:
+
+	Spider         ADXL345
+	3V3            VCC
+	GND            GND
+	MISO           SDO
+	MOSI           SDA
+	SCK            SCL
+	PA2            CS
```

Comment:
Still using 3.3V and PA2 here

Reply:
All the above sorted.

I just generalised the comment about the probe voltage rather than mention the BL-Touch.